vaadin-form-layout-flow-parent/vaadin-form-layout-testbench/src/main/java/com/vaadin/flow/component/formlayout/testbench/FormLayoutElement.java

```diff
@@ -24,4 +26,30 @@
  */
 @Element("vaadin-form-layout")
 public class FormLayoutElement extends TestBenchElement {
+
+    /**
+     * Retrieves all form items of type {@link FormItemElement}.
+     *
+     * @return a list of all {@link FormItemElement} instances.
+     */
+    public List<FormItemElement> getFormItems() {
+        return $(FormItemElement.class).all();
+    }
+
+    /**
+     * Retrieves a form item element by its label.
+     *
+     * @param label
+     *            the label of the form item to retrieve
+     * @return the form item element with the specified label
+     * @throws IllegalArgumentException
+     *             if no form item is found with the specified label
+     */
+    public FormItemElement getFormItemByLabel(String label) {
+        return getFormItems().stream().filter(item -> {
+            var itemLabel = item.getLabel();
+            return itemLabel != null && itemLabel.getText().equals(label);
+        }).findFirst().orElseThrow(() -> new IllegalArgumentException(
+                "No form item found with label: " + label));
```

Comment:
Same question here. What's the benefit of throwing an exception instead of returning `null`?

```suggestion
        return getFormItems().stream()
                .filter(item -> label.equals(item.getLabelText())).findFirst()
                .orElse(null);
```